paper-server/src/main/java/io/papermc/paper/configuration/WorldConfiguration.java

```diff
@@ -575,6 +575,7 @@ public class Misc extends ConfigurationPart {
         public int shieldBlockingDelay = 5;
         public boolean disableRelativeProjectileVelocity = false;
         public boolean legacyEnderPearlBehavior = false;
+        public boolean dragonSpawnAttemptPerformanceImprovements = true;
```

Comment:
I think allowRemoteEnderDragonRespawning would better describe what this option does (with the default set to false)